ocfweb/docs/docs/services/account/content-removal.md

```diff
@@ -0,0 +1,46 @@
+[[!meta title="Content Removal Policy"]]
+
+
```

Comment:
this should probably have a paragraph clarifying that most of the content on ocf.berkeley.edu is not the ocf's, owned by users, etc. that's usually the first point of confusion (think of the campanile stuff)